Assignment3/Q1.java

```diff
@@ -0,0 +1,148 @@
+import java.util.*;
+
+class Q1
+{
+	public static void main(String args[])
```

Comment:
You don't have to, but it is a useful skill to learn to use JUnit to test your code. http://junit.org/junit4/
This is used by almost every company that writes Java, and so is a super useful skill to demonstrate at interview. Knowing how to use a library like Truth https://github.com/google/truth is even better.